tests/CMakeLists.txt

```diff
@@ -56,6 +56,24 @@ if(ENABLE_BLACKMAGIC)
 	endif()
 endif()
 
+####################### OPENCV ##########################
```

Comment:
Same suggestion on all CmakeLists.txt files... let's rebase this file on 'develop', and then add back in these changes.... so we aren't reverting any updates.